perllib/FixMyStreet/Cobrand/Sutton.pm

```diff
@@ -738,6 +738,7 @@ sub bulky_allowed_property {
 
 sub bulky_collection_time { { hours => 6, minutes => 0 } }
 sub bulky_cancellation_cutoff_time { { hours => 6, minutes => 0, days_before => 0 } }
+sub small_items_cancellation_cutoff_time { { hours => 23, minutes => 59, days_before => 0 } }
```

Comment:
I don't understand why this isn't the same as bulky? Collections start at 6 so no point in cancelling after that.